docs/developers-clear-containers-install.md

```diff
@@ -0,0 +1,131 @@
+# Developers Clear Containers 3.0 Install
+
+This is not a end-user guide, this is for people that are eager to try Clear Containers 3.0 and want to build it from source and are familiar doing so. Basic programming knowlege  and be confortable to work with the Linux command line is expected.
+
+
+## Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html) Required to build **Virtcontainers pause**
+
+## Clear Containers 3.0 components
+
+  * [Runtime](https://github.com/clearcontainers/runtime)
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers) Both projects ship ``cc-proxy`` and they are not compatible with each other.
+
+There are two issues to enable ``--version`` option in ``Proxy`` and ``Shim`` so the end user can know for sure which version of these components he is running.
+
+  * [https://github.com/clearcontainers/proxy/issues/69](https://github.com/clearcontainers/proxy/issues/69)
+  * [https://github.com/clearcontainers/shim/issues/32](https://github.com/clearcontainers/shim/issues/32)
+
+## Setup the environment
+
+### Define GOPATH
```

Comment:
My previous comment regarding this headings still holds. Just change the headings to numbered list items. For example:
1. Define GOPATH: